client/src/styles/fonts.module.scss

```diff
@@ -0,0 +1,74 @@
+// Font for headers etc. is:'Sofia Sans'
+@import url('https://fonts.googleapis.com/css2?family=Sofia+Sans:ital,wght@0,700;0,800;0,900;1,700;1,800;1,900&display=swap');
+// Font for text blocks etc. is: 'Lato'
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,700;1,300;1,700&display=swap');
+
+$header-font: 'Sofia Sans', sans-serif;
+$body-font: 'Lato', 'Calibri', sans-serif;
+
+h1, h2, h3, h4, h5, h6 {
+  font-family: $header-font;
+}
+
+* {
+  font-family: $body-font;
+  font-size: 1rem;
+  font-weight: 300;
+}
+
+h1, h6 {
+  font-weight: 900;
+}
+
+h2, h5 {
+  font-weight: 800;
+}
+
+h3, h4 {
+  font-weight: 700;
+}
+
+h2, h4, h5 {
+  font-style: italic;
+}
+
+h1 {
+  font-size: 2.25rem;
+}
+
+h2 {
+  font-size: 1.8rem;
```

Comment:
Just fyi - using font sizes that are not dividable by 0,125 are quite rare and are an indicator of a weird design choice most of the time :D If your designer in a company ever wants you to code paddings like 39px or 15.5px font-sizes etc, don't do it but rather challenge it and ask wether you can round it up/down or wether he/she can update the designs. It's just a bad habit :D